lib/data/v2/system-env-variables.js

```diff
@@ -0,0 +1,169 @@
+import ProductName from '~/components/product-name'
+import Link from '~/components/text/link'
+
+export const BASE_SYSTEM_ENVS = [
+  {
+    name: 'NOW_URL',
+    description: 'The URL of the deployment.'
+  }
+]
+
+export const GITHUB_SYSTEM_ENVS = [
+  {
+    name: 'NOW_GITHUB_DEPLOYMENT',
+    description: (
+      <>
+        An indicator for whether the app was deployed by <ProductName /> for
+        GitHub.
+      </>
+    )
+  },
+  {
+    name: 'NOW_GITHUB_ORG',
+    description:
+      'The GitHub organization that owns the repository the deployment is triggered from.'
+  },
+  {
+    name: 'NOW_GITHUB_REPO',
+    description: 'The origin repository of the app.'
+  },
+  {
+    name: 'NOW_GITHUB_COMMIT_ORG',
+    description:
+      'The organization of which the commit belongs. For example, when submitting a pull request from a forked repository.'
+  },
+  {
+    name: 'NOW_GITHUB_COMMIT_REPO',
+    description:
+      'The repository of which the commit belongs. For example, when submitting a pull request from a forked repository.'
+  },
+  {
+    name: 'NOW_GITHUB_COMMIT_REF',
+    description: 'The branch that the deployment was made from.'
+  },
+  {
+    name: 'NOW_GITHUB_COMMIT_SHA',
+    description: (
+      <>
+        The{' '}
+        <Link href="https://help.github.com/articles/github-glossary/#commit">
+          sha
```

Comment:
```suggestion
          SHA
```

Its an acronym for Secure Hash Algorithms so it should be uppercase

Reply:
Nice catch 👍